### PR TITLE
Documenting  GaussianTransferFunction.cc

### PR DESCRIPTION
**Pull request by vidalm**
First try. As I said to Seb. W. I am not convince of the first paragraph. It is a copy paste from the Seb's binned TF. I do not have any other good idea to rephrased it (vacation mode) so...

### CODE REVIEW

**Comment by blinkseb** [comment]
More than enough IMO. We thanksfully don't write a novel :stuck_out_tongue_closed_eyes:

File: modules/GaussianTransferFunction.cc [hunk 1]
```diff
@@ -26,7 +26,34 @@
...
+ * and evaluates the transfer function on the "reconstructed" and "generated" energies.
+ *
+ * The transfer function (TF) is a Gaussian distribution that describes the difference between 
+ * the reconstructed and the generated energy (\f$E_{rec}-E_{gen}\f$). The width of the distribution depends 
+ * on the "reconstructed" energy and it is set as parameter. 
+ *
+ *  - Integration dimension: 1
+ *
+ *  - Parameters:
+ *    - `sigma` (double): Fraction of the "reconstructed" energy corresponding to the sigma of the Gaussian distribution.
```
Comment (swertz) [inline modules/GaussianTransferFunction.cc]: I'm sorry but I don't find this definition very clear :p

File: modules/GaussianTransferFunction.cc [hunk 2]
```diff
@@ -26,7 +26,34 @@
...
-/**
+/** \brief Transfer function on energy described by a Gaussian distribution
+ *
+ * This module takes as inputs a LorentzVector and a phase-space point, generates
+ * a new LorentzVector with a different energy (keeping direction and invariant mass),
+ * and evaluates the transfer function on the "reconstructed" and "generated" energies.
+ *
+ * The transfer function (TF) is a Gaussian distribution that describes the difference between 
+ * the reconstructed and the generated energy (\f$E_{rec}-E_{gen}\f$). The width of the distribution depends 
+ * on the "reconstructed" energy and it is set as parameter.
```
Comment (swertz) [inline modules/GaussianTransferFunction.cc]: I was just thinking, in principle the width should depend on the generated energy right? The resolution is usually expressed as a function of the true energy?

File: modules/GaussianTransferFunction.cc (hunk 1 above)
Comment (blinkseb) [inline modules/GaussianTransferFunction.cc]: isn't `width of the gaussian` enough?

Comment (swertz) [inline modules/GaussianTransferFunction.cc]: Well, the width is really `sigma*E`, right?

File: modules/GaussianTransferFunction.cc (hunk 2 above)
Comment (vidalm) [inline modules/GaussianTransferFunction.cc]: In principle yes, but it is hard to implement in this way. Actually this Gaussian TF is just to play. The right way would be a fitted TF where you know this info.

Comment (swertz) [inline modules/GaussianTransferFunction.cc]: True... There might be a way, but let's keep that for a more realistic TF module.

**Pull request merged**